frontend/src/static/js/components/styles/ManageItemList-filters.scss

```
@@ -133,18 +130,15 @@
 
 			button {
 				display: inline-block;
-				padding: 3px 6px 4px 0;
+				padding: 4px 8px;
 				// line-height:16px;
```

Stylelint (17.11.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)